src/ember-app.js

```diff
@@ -234,30 +203,13 @@ class EmberApp {
     return AppFactory['default']();
   }
 
-  /**
-   * @private
-   *
-   * Initializes the sandbox by evaluating the Ember app's JavaScript
-   * code, then retrieves the application factory from the sandbox and creates a new
-   * `Ember.Application`.
-   *
-   * @returns {Ember.Application} the Ember application from the sandbox
-   */
-  retrieveSandboxedApp() {
-    this.loadAppFiles();
-
-    return this.createEmberApp();
-  }
-
   /**
    * Destroys the app and its sandbox.
    */
   destroy() {
     if (this.app) {
       this.app.destroy();
     }
```

Comment:
I think `this.app` doesn't exists in the new changes any more so this condition will always be `false`.